ssh/ssh.go

```diff
@@ -0,0 +1,66 @@
+package ssh
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"golang.org/x/crypto/ssh"
+	"log"
+
+	"github.com/SUSE/scf-secret-generator/model"
+	"github.com/SUSE/scf-secret-generator/util"
+)
+
+type SSHKey struct {
+	PrivateKey  string // Name to associate with private key
+	Fingerprint string // Name to associate with fingerprint
+}
+
+func GenerateSSHKey(secretData map[string][]byte, key SSHKey) bool {
+	secretKey := util.ConvertNameToKey(key.PrivateKey)
+
+	// Only create keys, don't update them
+	if _, ok := secretData[secretKey]; !ok {
```

Comment:
Do an early return here and outdent the rest of the function:

```
if ...; ok {
    return false
}
```

Reply:
Fixed in 05a4ca2

I updated password.go as well just to be consistent.